crates/lakekeeper/src/service/authz/mod.rs

```diff
@@ -679,6 +689,18 @@ pub(crate) mod tests {
         pub(crate) fn hide(&self, object: &str) {
             self.hidden.write().unwrap().insert(object.to_string());
         }
+
+        fn action_is_blocked(&self, action: &str) -> bool {
+            self.blocked_actions.read().unwrap().contains(action)
+        }
+
+        // `namespace:can_list_everything`
```

Comment:
We should probably remove the comment.
Should we make `block_action` `pub(crate)`?